web-ui/src/main/resources/ui/app/Pie.js

```diff
@@ -0,0 +1,21 @@
+define(function(require) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["define" is not defined.](https://www.codacy.com/app/frank_7/spawncamping-dds/file/2194492813/issues/source?bid=3046234&fileBranchId=3126677#l1)